src/consumer/bit-map/bit-map.js

```diff
@@ -68,7 +75,7 @@ export default class BitMap {
       logger.info(`bit.map: unable to find an existing ${BIT_MAP} file. Will probably create a new one if needed`);
     }
 
-    return new BitMap(dirPath, mapPath, components, version);
+    return new BitMap(dirPath, mapPath, components, version || BIT_VERSION);
```

Comment:
remove the else in the condition above.
put the return also inside the fs.existsSync(mapPath), and inside the id use the return without the || BIT_VERSION, in order to make sure migration will run for old bitmaps